src/ImageComparator.php

```diff
@@ -347,4 +352,62 @@ private function processImagePixels(
 
         return $pixels;
     }
+
+    public function calculateRGBDistance($color1, $color2): float
```

Comment:
@D4lv1k let's add type-hints for function parameters. Also, shouldn't this function have `private` visibility?